File: submissions/sm_035_yashwanth/week_3/day_4/session1/mirrorImage.js
var capital ="ABCDEFGHIJKLMNOPQRSTUVWXYZ"
var small ="abcdefghijklmnopqrstuvwxyz"
var str =""
function mirror(string){
    for (i = 0;i< string.length; i++) {
        for (j = 0; j<capital.length ; j++) {
         if(string[i] == capital[j] ){
             str += capital[25-j]
         }
         else if(string[i] == small[j]) {
          str += small[25-j]
         }       
        }
    }
    console.log(str)
}
mirror("AyzB")

File: submissions/sm_107_hasmuddin/week_13/day_4/session_1/mirror_image.py
str = str(input("mirror str: "))
small="abcdefghijklmnopqrstuvwxyz"
caps = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
mirror = ""
for st in range(len(str)):
    if (str[st].isupper()):
        ind = caps.index(str[st])
        mirrIndex = 25 - ind
        mirror += caps[mirrIndex]
    else:
        ind = small.index(str[st])
        mirrIndex = 25 - ind
        mirror += small[mirrIndex]

print(mirror)
